includes/admin/payments/orders.php

```diff
@@ -742,33 +794,60 @@ function edd_order_details_extras( $order = false ) {
 			<div class="edd-admin-box">
 				<?php do_action( 'edd_view_order_details_payment_meta_before', $order->id ); ?>
 
-				<?php if ( ! edd_is_add_order_page() && ! empty( $order->id ) ) : ?>
-					<?php if ( ! empty( $order->gateway ) ) : ?>
+
+				<?php if ( ! edd_is_add_order_page() ) : ?>
 					<div class="edd-order-gateway edd-admin-box-inside edd-admin-box-inside--row">
 						<span class="label"><?php esc_html_e( 'Gateway', 'easy-digital-downloads' ); ?></span>
 						<span class="value"><?php echo edd_get_gateway_admin_label( $order->gateway ); ?></span>
 					</div>
-					<?php endif; ?>
 				<?php else : ?>
 					<div class="edd-order-gateway edd-admin-box-inside">
-						<span class="label"><?php esc_html_e( 'Gateway', 'easy-digital-downloads' ); ?></span>
-						<?php echo $gateway_select; ?>
+						<div class="edd-form-group">
+							<label for="edd_gateway_select" class="edd-form-group__label"><?php esc_html_e( 'Gateway', 'easy-digital-downloads' ); ?></label>
+							<div class="edd-form-group__control">
+								<?php
+								echo EDD()->html->select(
+									array(
+										'name'             => 'gateway',
+										'class'            => 'edd-form-group__input',
+										'id'               => 'edd_gateway_select',
+										'options'          => $gateways,
+										'selected'         => edd_get_default_gateway(),
+										'chosen'           => true,
+										'show_option_none' => false,
+										'show_option_all'  => false,
+									)
+								); // WPCS: XSS ok.
+								?>
+							</div>
+						</div>
 					</div>
 				<?php endif; ?>
 
-				<div class="edd-order-payment-key edd-admin-box-inside">
-					<label for="edd_payment_key" class="label"><?php esc_html_e( 'Key', 'easy-digital-downloads' ); ?></label>
-					<input type="text" name="payment_key" id="edd_payment_key" <?php echo esc_attr( $readonly ); ?> value="<?php echo esc_attr( $order->payment_key ); ?>" />
+				<div class="edd-admin-box-inside">
+					<div class="edd-form-group">
+						<label for="edd_payment_key" class="edd-form-group__label"><?php esc_html_e( 'Key', 'easy-digital-downloads' ); ?></label>
+						<div class="edd-form-group__control">
+							<input type="text" name="payment_key" id="edd_payment_key" class="edd-form-group__input" <?php echo esc_attr( $readonly ); ?> value="<?php echo esc_attr( $order->payment_key ); ?>" />
```

Comment:
If we could go ahead and add 'regular-text` to this as a class I would like that. Just so I can see the entire key.